pyfar/dsp/interpolation.py

```diff
@@ -495,6 +496,14 @@ def resample(signal, sampling_rate, match_amplitude="auto", frac_limit=None):
         realizing the target sampling rate.
 
         The default is ``None``, which uses ``frac_limit = 1e6``.
+    suppress_aliasing : bool, optional
+        Suppress aliased energy above the Nyquist frequency of the input
+        signal, i.e., ``signal.sampling_rate/2``. This is recommended, and is
+        realized by applying a zero-phase Elliptic filter with a pass band
+        ripple of 0.1 dB, a stop band attenuation of 60 dB, and pass/stop
+        band frequencies of ``signal.sampling_rate/2`` and
+        ``min(1, 1.05 * signal.sampling_rate/2)`` Hz. Note that this is only
```

Comment:
This is a little confusing.. it just says that the passband edge frequency is the old Nyquist frequency and the stopband edge frequency is the minimum of (new Nyquist frequency/1.2*old Nyquist frequency), right?

Reply:
Yes, I tried to make that more clear